src/aap_eda/settings/default.py

```diff
@@ -823,3 +818,9 @@ def get_rulebook_process_log_level() -> RulebookProcessLogLevel:
 MAX_PG_NOTIFY_MESSAGE_SIZE = int(
     settings.get("MAX_PG_NOTIFY_MESSAGE_SIZE", 6144)
 )
+
+DEFAULT_SYSTEM_PG_NOTIFY_CREDENTIAL_NAME = "_DEFAULT_EDA_PG_NOTIFY_CREDS"
+PGSSLMODE = settings.get("PGSSLMODE", default="prefer")
```

Comment:
As I point out in my other comment, we should not need to re-define this setting since `PGSSLMODE` is read for `databases` in ` _get_databases_settings()`

Reply:
@Alex-Izquierdo Done